assets/js/base/components/drawer/style.scss

```
@@ -72,8 +75,8 @@ $drawer-width-mobile: 100vw;
 	position: fixed;
 	right: 0;
 	top: 0;
-	transform: translateX(-$drawer-width);
-	width: $drawer-width;
+	transform: translateX(var(--neg-drawer-width));
+	width: var(--drawer-width);
```

Would something like this fix the issue? 
```suggestion
	transform: translateX(max(-100%, var(--neg-drawer-width)));
	width: var(--drawer-width);
	max-width: 100%;
```